### PR TITLE
Shipping and payment details

### PR DESCRIPTION
**Pull request by ds300**
https://artsyproduct.atlassian.net/browse/PURCHASE-311

https://zpl.io/25qjjq3

This PR shares a commit with #1091 – How do we handle stacked PRs here? It doesn't seem to be doable when working with forks.

image

Need to update the icon set to match https://app.zeplin.io/project/5acd19ff49a1429169c3128b/screen/5acf57cbc958883984ec6abd or use stripe's icons.

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 0aed428c2476ef9a49ec6c75657f09e8cd770b30

https://deploy-preview-1092--artsy-reaction.netlify.com

**Comment by sweir27** [comment]
Your PRs should be okay as long as there aren't any conflicts-- do you care which is merged first?

**Comment by ds300** [comment]
Not in terms of correctness, It's just that the shared stuff is not trivial
to review and I want to avoid wasting people's time.

File: src/Styleguide/Elements/icons/CreditCardIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+      <use fill="#EFEFEF" xlinkHref="#a" />
+      <rect width={29} height={19} x={0.5} y={0.5} stroke="#E5E5E5" rx={2} />
+      <g fill="#4DB3FF">
+        <path d="M9.999 11A.999.999 0 0 0 9 12.01v1.98A1 1 0 0 0 9.999 15H30v-4H9.999zM0 6v4h20.001A.999.999 0 0 0 21 8.99V7.01A.999.999 0 0 0 20.001 6H0z" />
+      </g>
+    </g>
+  </svg>
+)
+
+const DiscoverIcon = (props: React.SVGProps<SVGSVGElement>) => (
```
Comment (sweir27) [inline src/Styleguide/Elements/icons/CreditCardIcon.tsx]: I would have expected these to live in separate files but it's actually kind of nice/simple to list them all here.

**Comment by sweir27** [comment]
This looks good, gonna merge and hopefully this doesn't require a rebase of the other PR. 😄

**Pull request merged**